example-app/src/main/java/com/adyen/checkout/example/ui/blik/BlikActivity.kt

```diff
@@ -68,28 +70,20 @@ class BlikActivity : AppCompatActivity() {
             BlikViewState.Loading -> {
                 binding.progressIndicator.isVisible = true
                 binding.errorView.isVisible = false
-                binding.blikView.isVisible = false
+                binding.componentView.isVisible = false
             }
             is BlikViewState.ShowComponent -> {
                 binding.progressIndicator.isVisible = false
                 binding.errorView.isVisible = false
-                binding.blikView.isVisible = true
+                binding.componentView.isVisible = true
 
                 setupBlikView(blikViewState.paymentMethod)
             }
             is BlikViewState.Await -> {
```

Comment:
This should be changed to `BlikViewState.Action` and the `Unsupported` event should be removed